test/Sleep-test.js

```diff
@@ -28,4 +28,9 @@ describe('Sleep', () => {
   it('should be able return a week\'s worth of sleep data for a user', () => {
     expect(sleep.returnWeekOfSleepInfo(1, 'sleepQuality')).to.eql([1.2, 1.2, 4.2, 3, 1.5, 1.3, 3.7]);
   });
+
+  it('should test inheritance', () => {
+    expect(sleep.returnIndividualStatForDate('2019/06/15', 'hoursSlept')).to.equal(6.1);
+    expect(sleep.returnIndividualStatForDate('2019/06/24', 'sleepQuality')).to.equal(1.3);
+  });
```

Comment:
Please see comment in activity-test about the feedback from Brittany indicating we do not need to test in this way.